coderd/templates_test.go

```diff
@@ -635,6 +635,7 @@ func TestPatchTemplateMeta(t *testing.T) {
 		const (
 			failureTTL    = 7 * 24 * time.Hour
 			inactivityTTL = 180 * 24 * time.Hour
+			lockedTTL     = 360 * 24 * time.Hour
```

Comment:
NIT: I think I stuck my tests in the `coderd` folder as well - I'm wondering if they should actually live in `enterprise/coderd`

Reply:
probably good to have tests in both